data/README.md

```diff
@@ -38,8 +38,13 @@
 | datapage_v1-snappy-compressed-checksum.parquet | compressed INT32 columns in v1 data pages with a matching CRC          |
 | datapage_v1-corrupt-checksum.parquet           | uncompressed INT32 columns in v1 data pages with a mismatching CRC     |
 | overflow_i16_page_cnt.parquet                  | row group with more than INT16_MAX pages                   |
+<<<<<<< HEAD
```

Comment:
Hmm, can you remove the merge markers here?

Reply:
🤦